test/system/TestVarQualifiedNamedUsed.hs

```diff
@@ -1,9 +1,9 @@
 -- 
--- import qualified System.IO as IO (putStrLn)
```

Comment:
Such change is not preferable. Because `import qualified System.IO as IO (putStrLn)` and `import qualified System.IO as IO` has really different semantic. Removing imports from `as`-lists should be as follows:

```haskell
-- print
-- import qualified System.IO as IO (putStrLn)

module TestVarQualifiedNamedOnlyUsed where

import qualified System.IO as IO (putStrLn, print)

main :: IO ()
main = IO.putStrLn "test"
```